TruckMuncher/MapViewController.swift

```diff
@@ -34,7 +50,39 @@ class MapViewController: UIViewController, MKMapViewDelegate, CLLocationManagerD
         mapClusterController.delegate = self
         setClusterSettings()
         
-        updateData()
+        truckCarousel = iCarousel()
+        truckCarousel.type = .Linear
+        truckCarousel.delegate = self
+        truckCarousel.dataSource = self
+        truckCarousel.pagingEnabled = true
+        truckCarousel.currentItemIndex = 0
+        truckCarousel.bounces = false
+        
+        view.addSubview(truckCarousel)
+        attachGestureRecognizerToCarousel()
+        
+        var muncherImage = UIImage(named: "truckmuncher")
+        var muncherImageView = UIImageView(image: muncherImage)
+        var navFrame = navigationController?.navigationBar.frame
+        muncherImageView.frame = CGRectMake(navFrame!.midX - 20.0, 0.0, 40.0, 40.0)
+        navigationController?.navigationBar.addSubview(muncherImageView)
+        
+        navigationItem.rightBarButtonItem = UIBarButtonItem(title: "Sign In", style: .Plain, target: self, action: "login")
+    }
+    
+    override func viewDidAppear(animated: Bool) {
+        super.viewDidAppear(animated)
+        truckCarousel.frame = CGRectMake(0.0, mapView.frame.maxY - 100.0, UIScreen.mainScreen().bounds.width, UIScreen.mainScreen().bounds.height)
+
+        menuManager.getFullMenus(atLatitude: 0, longitude: 0, includeAvailability: true, success: { (response) -> () in
```

Comment:
eventually we will want to move all of this into a method that gets called after the location is available. Then we can use the location in the request.